### ISSUE TITLE
bug(mimic, executioner): if you mimic an executioner you won't know their target

### ISSUE THREAD

**Issue by pikulet** [r3ndd/BeyondMafia-Integration#291]
need to check this for amnes/maid too

**Comment by DrSharky** [comment]
Well the bug should be that mimic actually becomes executioner. Executioner is 3rd party, the mimic should become amnesiac instead.

**Comment by pikulet** [comment]
Good point. This bug was about how mimic does not go through the "rolesAssigned" listener used by executioner.
